package.json

```diff
@@ -8,6 +8,7 @@
     "@openzeppelin/contracts": "^4.4.2",
```

Comment:
@xiangformless Can you pin these other packages as well? eg. remove the ^

Reply:
![image](https://github.com/user-attachments/assets/c7089fb6-e28d-44e3-a4c7-400b5341137b)
is this OK? or pin all of them?

Reply:
@un7c0rn